library/core/src/intrinsics/mod.rs

```diff
@@ -3965,8 +3981,16 @@ pub const fn is_val_statically_known<T: Copy>(_arg: T) -> bool {
 /// The stabilized form of this intrinsic is [`crate::mem::swap`].
 ///
 /// # Safety
+/// Behavior is undefined if any of the following conditions are violated:
```

Comment:
This is incomplete. The operation is typed, so the pointers must point to data that is valid at the given type.